src/App.tsx

```diff
@@ -1,17 +1,78 @@
 import { useTranslation } from 'react-i18next'
+import LangSwitcher from './components/LangSwitcher/LangSwitcher'
+import logo from './assets/logo.svg'
+import gifAdult from './assets/bg-animation.gif'
+import gifCh from './assets/bg-animation-ch.gif'
+import flowers from './assets/flowers.svg'
+import flowersMobile from './assets/flowers-2.svg'
+import './scss/sections/hero.scss'
 
 function App() {
-	const { t } = useTranslation()
+	const { t, i18n } = useTranslation()
 
 	return (
 		<>
-			<h1>Vite + React</h1>
-			<div className="card">
-				<h2 className="t__h2">{t('heroTitle')}</h2>
-			</div>
-			<div>
-				<p>ПОРУЧ ДІТИ</p>
-			</div>
+			<section className="hero">
+				{i18n.language === 'ch' && (
+					<img
+						src={window.innerWidth <= 375 ? flowersMobile : flowers}
+						className={window.innerWidth <= 375 ? 'hero__bg-flowers-mobile' : 'hero__bg-flowers'}
+					/>
+				)}
+				<img
+					src={i18n.language === 'ad' ? gifAdult : gifCh}
```

Comment:
why is there a condition here instead of using a method with a variable

Reply:
UPD

![image](https://github.com/BUI-studies/buoy-landing/assets/109099364/9ee58440-6d6d-46ca-a7e2-b34f6e99eb1f)